components/OHPC_macros

```diff
@@ -75,21 +75,24 @@ BuildRequires: ohpc-buildroot
 # these choices can be overridden by specifing the compiler_family/mpi_family
 # variables via rpmbuild or other mechanisms.
 
-%{!?compiler_family: %global compiler_family gnu9}
+%{!?compiler_family: %global compiler_family gnu12}
 %{!?mpi_family: %global mpi_family openmpi4}
 %{!?python_family: %global python_family python3}
 
 # Compiler dependencies
 %if 0%{?ohpc_compiler_dependent} == 1
 
+%if "%{compiler_family}" == "gnu12"
+BuildRequires: gnu12-compilers%{PROJ_DELIM} >= 12.1.0
+Requires:      gnu12-compilers%{PROJ_DELIM} >= 12.1.0
+%global gnu_family gnu12
+%endif # gnu9
```

Comment:
`#gnu12`

Reply:
Good catch. I also need to move it off the same line. The RPM developers have indicated they may turn end-of-line comments into errors.